Development/#3 Post form from React frontend to Django Restful API-20210930T183306Z-001/#3 Post form from React frontend to Django Restful API/frontend/src/components/leadFormContainerView.jsx

```diff
@@ -18,7 +18,13 @@ class LeadFormContainerView extends Component {
     };
   }
 
-  handleSubmit() {}
+  handleSubmit() {

```

Comment:
Could you format your code using an ESLinter?

Reply:
Done please review